common/changes/@bentley/imodelbank-client/benp-add-rush-cache_2022-08-25-18-02.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@bentley/imodelbank-client",
+      "comment": "add rush-project.json",
```

Comment:
you can actually set the comment for all of these rush changelogs to an empty string;
these are what gets published as part of the pkg changelog, and this change shouldnt really be mentioned in it

Reply:
Wondered about that - thanks.

Reply:
Done